specs/promise-cacheable.decorator.spec.ts

```diff
@@ -511,6 +512,68 @@ strategies.forEach(s => {
       expect(mockServiceCallSpy).toHaveBeenCalledTimes(3);
     });
 
+    it('only cache data when a specific response (set from globalConfig) is returned, otherwise it should bail to service call', async () => {
```

Comment:
@angelnikolov After adding these specs which manipulate ```GlobalCacheConfig``` the specs fail randomly. Can you advise how to fix this?  The same specs in ```observable-cacheable.decorator.spec.ts``` seem to test fine.